tokio/src/blocking.rs

```diff
@@ -16,7 +22,16 @@ cfg_not_rt! {
     {
         assert_send_sync::<JoinHandle<std::cell::Cell<()>>>();
         panic!("requires the `rt` Tokio feature flag")
+    }
 
+    cfg_fs! {
+        pub(crate) fn spawn_mandatory_blocking<F, R>(_f: F) -> JoinHandle<R>
+        where
+            F: FnOnce() -> R + Send + 'static,
+        R: Send + 'static,
```

Comment:
Rustfmt doesn't look inside macro invocations.
```suggestion
            F: FnOnce() -> R + Send + 'static,
            R: Send + 'static,
```